### ISSUE TITLE
Draw a gradient descent concept diagram.

### ISSUE THREAD

**Issue by xuyuji9000** [xuyuji9000/machine-learning-playground#44]

**Comment by xuyuji9000** [comment]
draw.io diagram documented.

It is a diagram about the relationship between loss function, gradient, gradient descent and backpropagation.

Closing this issue.
